server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/SetMessagesMethodStepdefs.java

```diff
@@ -135,4 +138,18 @@ public void setFlags(String message, List<String> keywords) throws Throwable {
             .discardContent();
         mainStepdefs.awaitMethod.run();
     }
+
+    @When("^the user set flags (.*) via messageIdProbe of \"([^\"]*)\" on \"([^\"]*)\" of user \"([^\"]*)\"")
```

Comment:
Well, here too, the jerkin spec is not good enough.

I propose "Message ".\*" has flags ".\*" in mailbox ".\*" of user ".\*""